src/core/ext/xds/xds_bootstrap.cc

```diff
@@ -29,7 +29,7 @@ namespace grpc_core {
 // removed once federation is fully integrated and enabled by default.
```

Comment:
Historically, in C-core, we've just completely removed the env var instead of changing the default to true, on the theory that once we pass interop tests, we have confidence that there won't be any more need to disable the feature.  However, in this particular case, since we've already had a fairly serious bug with this code, it probably does make sense to be more cautious here, so I'm okay with this approach.

However, we need to remember to eventually come back and remove this completely, or else we'll just keep accumulating env var controls that we don't actually need.  Please update the TODO here to indicate that this should be removed after the 1.55 release, just so that we have a target date for removing this.

Reply:
done, updated the todo